examples/trials/efficientnet/README.md

```diff
@@ -0,0 +1,22 @@
+# EfficientNet
+
+[EfficientNet: Rethinking Model Scaling for Convolutional Neural Networks](https://arxiv.org/abs/1905.11946)
+
+Provided here are:
+
+* Search space and tuners for finding the best tuple (alpha, beta, gamma) for EfficientNet-B1 with grid search, as discussed in Section 3.3 in [paper](https://arxiv.org/abs/1905.11946).
+* Search space for EfficientNet transfer learning on datasets other than ImageNet; settings are borrowed from Appendix A.5 from "[Do Better ImageNet Models Transfer Better?](https://arxiv.org/abs/1805.08974)".
+
```

Comment:
It seems the work in second paper (https://arxiv.org/abs/1805.08974)  is not specific to EfficientNet.   What we provided in this example for 2nd paper is just learning rate search (which we already have a lot of examples).
Can we remove the second part?  so that this exmaple will more concise and clearer.

Reply:
The 2nd paper is cited by EfficientNet, for the settings of transfer learning. And yes, it's not important in the big picture. The main reason I show this here is to provide an example on how to use transfer learning with EfficientNet-PyTorch. I'm fine with removing this part. What about @QuanluZhang ?